src/components/DryRunnerCardLog/index.js

```diff
@@ -23,27 +25,30 @@ const DryRunnerCardLog = React.memo(props => {
       <div className='DryRunnerCardLog__container'>
         <Row>
           {cards.map((card, index) => (
-            <Column
-              style={{ opacity: 1 - index / 8 }}
-              width='1/6'
-              key={(card ? card.id : '') + '#' + index}
-            >
-              {card && (
-                <Image
-                  wrapperClassName='DryRunnerCardLog__image-wrapper'
-                  className='DryRunnerCardLog__image'
-                  src={card.image}
-                  alt={card.name}
-                  onClick={() => setZoomedCard(card)}
-                  data-testid='card-log-image'
-                />
+            <>
+              <Column
+                style={{ opacity: 1 - index / 8 }}
+                width='1/6'
+                key={(card ? card.id : '') + '#' + index}
+              >
+                {card && (
+                  <Image
+                    wrapperClassName='DryRunnerCardLog__image-wrapper'
+                    className={'DryRunnerCardLog__image'}
+                    src={card.image}
+                    alt={card.name}
+                    onClick={() => setZoomedCard(card)}
+                    data-testid='card-log-image'
+                  />
+                )}
+              </Column>
+              {props.cardsThisTurn === index + 1 && index !== 5 && (
+                <CardLogSeparator />
               )}
```

Comment:
I don’t think inserting an extra element alongside the columns is necessary. I think we should be able to apply an extra class to all cards that are the first in a turn. Something like:

```js
<Image
  wrapperClassName={[
    'DryRunnerCardLog__image-wrapper',
    props.cardsThisTurn === index + 1 &&
      index !== 5 &&
      'DryRunnerCardLog__image-wrapper--turn',
  ].filter(Boolean)}
  className='DryRunnerCardLog__image'
  src={card.image}
  alt={card.name}
  onClick={() => setZoomedCard(card)}
  data-testid='card-log-image'
/>
```

Then on the CSS:

```css
.DryRunnerCardLog__image-wrapper--turn::after {
  content: '';
  position: absolute;
  right: -0.5em;
  height: 80%;
  top: 50%;
  width: 1px;
  transform: translateY(-50%);
  background: linear-gradient(
    to bottom,
    transparent,
    var(--dark-beige),
    transparent
  );
```

Reply:
Oh, and for the pseudo element to be able to be absolutely positioned, `.DryRunnerCardLog__image-wrapper` needs to be `position: relative`.

Reply:
Great it looks amazing!!